packages/grafana-ui/src/components/AlphaNotice/AlphaNotice.tsx

```diff
@@ -10,7 +10,9 @@ export interface Props {
   className?: string;
 }
 
+/** @deprecated Will be removed  in a future release */
 export const AlphaNotice: FC<Props> = ({ state, text, className }) => {
```

Comment:
Maybe we can just remove it straight away? I doubt it's used externally (maybe enterprise...)